Clarify AWS IAM Role Setup (#1056)
astronomer/docs: astro/connect-external-services.md

>     ]

Reviewer: Can't give suggestions, but a few notes on the earlier steps:

- When you access the IAM dashboard in step 3, does it have to be under an account that has access to the AWS service? If so, we could specify that
- Why do we use `AirflowS3Logs-<cluster-ID>` as the role specifically? Is it worth mentioning that this is where worker Pods run?

}
```

> :::info
> 
> Astronomer creates the `AirflowLogsS3-<clusterid>` role in the data plane account.  Every Airflow deployment in a cluster runs under this role.
> 
> :::

Reviewer: ```suggestion
    Your Astro cluster's data plane account includes the `AirflowLogsS3-<clusterid>` role that you just specified. When you configure an Airflow connection for a Deployment, specify this role in an [AWS Airflow Connection](https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/connections/aws.html) to provide your Deployment access to your service.
```

Is this correct? Is the role already created, or is it created after you

> By default, Astronomer creates the `AirflowLogsS3-<clusterid>` role in the data plane account.  Every Airflow deployment in a cluster runs under this role.

Reviewer 2: ```suggestion
Astronomer creates the `AirflowLogsS3-<clusterid>` role in the data plane account.  Every Airflow deployment in a cluster runs under this role.
```

Reviewer 2: Removed the By Default, because its not just the default behavior, its the only behavior. You can't override this.

7. Click **Update policy**.

> 8. In the Airflow UI or as an environment variable on Astro, create an Airflow connection to AWS for each Deployment that requires the resources you connected. See [Managing connections to Apache Airflow](https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/connections/aws.html).

Reviewer: ```suggestion
8. In the Airflow UI or as an environment variable on Astro, create an Airflow connection to AWS for each Deployment that requires the resources you connected, making sure to configure `role_arn` as `arn:aws:iam::<dataplane-AWS-account-ID>:role/AirflowS3Logs-<cluster-ID>`. See [Managing connections to Apache Airflow](https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/connections/aws.html).
```
